### PR TITLE
Add MLP tutorial and exercise

### PR DESCRIPTION
**Pull request by KianYang-Lee**
Add MLP tutorial and exercise

### CODE REVIEW

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb [hunk 1]
```diff
@@ -0,0 +1,1693 @@
...
+   "source": [
+    "# declare batch size\n",
+    "batch_size = 4\n",
+    "\n",
+    "# convert into Dataset using customized class\n",
+    "train_Dataset = ShampooDataset(X_sequence_train, y_sequence_train)\n",
+    "test_Dataset = ShampooDataset(X_sequence_test, y_sequence_test)\n",
+    "\n",
+    "# convert into DataLoader\n",
+    "train_loader = DataLoader(train_Dataset, batch_size=batch_size, shuffle=True)\n",
```
Comment (evanyeohboonkhai) [inline notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:629]: should be shuffle=False because we don want to break the information for the sequence

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb [hunk 2]
```diff
@@ -0,0 +1,1693 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# setting hyperparameter\n",
+    "input_size = 3\n",
```
Comment (evanyeohboonkhai) [inline notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:679]: input size  =  3 is because the window sequence = 3 ? might need to add a comment on this

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb [hunk 3]
```diff
@@ -0,0 +1,1693 @@
...
+    "    for i, (X, y) in enumerate(DataLoader(test_Dataset, 1)):\n",
+    "        count += 1\n",
+    "        output = model(X)\n",
+    "        predictions.append(output.item())\n",
+    "        \n",
+    "# convert list into np array and inverse transform it into proper values before feature scaling\n",
+    "predictions = np.array(predictions)\n",
+    "predictions = scaler.inverse_transform(predictions.reshape(1,-1))\n",
+    "\n",
+    "# remove beginning 3 rows of data since at least 3 data are needed to make a prediction\n",
```
Comment (evanyeohboonkhai) [inline notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:817]: why need to remove the beginning of 3 rows? because of the window size?

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb [hunk 4]
```diff
@@ -0,0 +1,1693 @@
...
...+R5FRfLXv3+wQyMb5s51xJYgQcOIFyZB9aFDUlLcUFAArFsHfO1rue07F9znY329fHcTGkgIObXoTNB4AMASpdSjkEonLwOAUmoKJOyEEEIipbc5NJ58Uu6mXXGF/+uZQk4qKmRwWlYGvPmmrIvboTF8uAyea2slhvvgwewdGoMGiZV8/frMFU4A/5ATpXK/29gd/H4Lb8gJIb2FTIJC2PhVQsoU8nLddcCXv5zdtl97DfjZz4BbbgE++UkJ3zNVmwxBd/wzhZyYUtKdCRrNzdJvegUNQESR3bsdcSabkBOvS86E9nXVoWHCToBgQWPMGGDaNODZZ+V5XV36fkaNSi3bHTZeh0Z5eXQVrwghdskoaGitbwPwNQC/B3C+1lq7PvfFaJtGCCEyUe7fXx7HLWj4OTT87MRdRWvgIx8BHnvMWffkk8A556TezXIT5NAoLJQBo1Li0jhxQo5bFJPr0lLZj9uh4c5iP2KEtOnAAfmOuTg0ABn8ZuvQcIecDB2aOUwlbBhyQohDnDk0/BwapaXAyZMStudl507JTZQNn/mMCMH/8R/AJZeIMGscb4Ygh0aQoNHUJP3TkCGdh5xs3y5Lb8gJ4FSxMtembEJOTAUsw6BB3XNoAMDnPy8hOZlyblx5JZBIyHXbG3ISB94cGuyXCTl16TTvvNb6Da31I1rrVte6LVprnzRJhBASLnV1MhgrKIg/5MTr0AiyE3eV3buB//1fuRPY1CQDzxUrMieyDMqh4a4kYsJOxo2L5o6UUhJ24haYjJNm6FD5vfbvd+5E5pJDw9CZoOHn0Igz3MSvDQAdGqT3EnfISUGBk1zS7B9IF521FgHClIPOREODhEJ87nPAaacBixdLKWpv2EkmQSOobOuAAY7Ym4klS2R59tnpr40fL9/d7D8Mh0ZXJvoTJgD33efkyvDjiivkmr10qR1Bo08fEaM+8Qngr3/1rxpGCDk1iKj6MyGEhIOJvfVOoOMg6hway5fLcv9+4Ec/kmSggFTGCCIo5MR9B84taERF377+Do3Bg2XgWFsrA2mga4JGtiEnbodG3IKG97c4dkz+eCeQ9EbCCDl5/nng2992xOQgqqulX3FXAfETGAEJ4Whrc8LgMrF+vSxnz5ZlWRlw7rnZCxqZcmgMGOD0jZl46ikpR+sXcjJ+vHyHjRvleWcOjY4Oub4EOTQGDUo9hmFy
```
Comment (evanyeohboonkhai) [inline notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:1403]: shuffle = false

**Review by evanyeohboonkhai — COMMENTED** [review]
Overall great job and good to read. Thanks

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb (hunk 3 above)
Comment (KianYang-Lee) [reply notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:817]: Beginning 3 rows are NA

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb (hunk 1 above)
Comment (KianYang-Lee) [reply notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:629]: What's breaking sequence information if one sequence only 3 data?

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb (hunk 4 above)
Comment (KianYang-Lee) [reply notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:1403]: What's breaking sequence information if one sequence only 3 data?

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb (hunk 2 above)
Comment (KianYang-Lee) [reply notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:679]: OK

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb (hunk 1 above)
Comment (evanyeohboonkhai) [reply notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:629]: yes, your point is correct, if not mistaken, you are breaking the sequence information between batches of data.For example 
data_list - [1 2 3 4 5 ]
Batch 1 - [1 2 3]
Batch 2 - [2 3 4]
Batch 3 - [2 4 5]
if shuffle =  False, the batch data will be arranged in Batch 1->Batch 2-> Batch 3
if shuffle = True, the batch data will be arranged in Batch 1 -> Batch 3 ->Batch 2

Comment (KianYang-Lee) [reply notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:629]: As per our discussion, resolving this

File: notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb (hunk 4 above)
Comment (KianYang-Lee) [reply notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb:1405]: As per our discussion, resolving this

**Review by evanyeohboonkhai — APPROVED** [review]
